### PR TITLE
fix(klaviyo): skip profile lookup call for rETL events

### PR DESCRIPTION
**Pull request by ujjwal-ab**
## Description of the change

Skip profile lookup call to check for existing user for rETL events

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by ItsSudip — APPROVED** [review]

File: src/v0/destinations/klaviyo/transform.js [hunk 1]
```diff
@@ -56,7 +56,10 @@ const identifyRequestHandler = async (message, category, destination) => {
   }
   const traitsInfo = getFieldValueFromMessage(message, 'traits');
   const response = defaultRequestConfig();
-  const personId = await isProfileExist(message, destination);
+  let personId;
+  if (!mappedToDestination) {
```
Comment (vyeshwanth) [inline src/v0/destinations/klaviyo/transform.js:60]: can we change this to contrext.channel == 'sources'. we have vdm and json mapper as well. this flag only works if retl connection is done by visual mapper

Comment (ujjwal-ab) [reply src/v0/destinations/klaviyo/transform.js:60]: Will this `channel === 'sources'` check only be sufficient or do we need both checks?

Comment (vyeshwanth) [reply src/v0/destinations/klaviyo/transform.js:60]: just this will be sufficient

Comment (ujjwal-ab) [reply src/v0/destinations/klaviyo/transform.js]: Updated

**Review by ItsSudip — APPROVED** [review]

**Review by vyeshwanth — APPROVED** [review]
LGTM

**Review by vyeshwanth — APPROVED** [review]
LGTM.

**Pull request merged**